### ISSUE TITLE
[docs] Sass and CSS guide boilerplate differences

### ISSUE THREAD

**Issue by florianbouvot** [foundation/foundation-emails#514]
Boilerplate in Sass and CSS docs doesn't use same classes:

`float-center` in [CSS guide](https://github.com/zurb/foundation-emails/blob/develop/docs/pages/css-guide.md#boilerplate)

```
<td class="float-center" align="center" valign="top">
  <center>
    <!-- The content of your email goes here. -->
  </center>
</td>
```

`center` in [Sass guide](https://github.com/zurb/foundation-emails/blob/develop/docs/pages/sass-guide.md#boilerplate)

```
<td class="center" align="center" valign="top">
  <center>
    <!-- The body of each email you write is injected here -->
      {{> body}}
  </center>
</td>
```

_`center` seems not to exist in Sass source_

Is that a mistake?

**Comment by rafibomb** [comment]
Yup - that's another good catch! Can you help update that with a PR?

**Comment by florianbouvot** [comment]
@rafibomb PR submit.

**Comment by DavidCao22** [comment]
@rafibomb Looks like the issue can be closed since it was fixed in #520
